srcpkgs/firejail/template

```diff
@@ -1,13 +1,15 @@
 # Template file for 'firejail'
 pkgname=firejail
 version=0.9.54
-revision=1
+revision=2
 build_style=gnu-configure
+makedepends="libapparmor-devel"
+configure_args="--enable-apparmor"
 short_desc="SUID security sandbox program"
 maintainer="Duncaen <duncaen@voidlinux.eu>"
 license="GPL-2.0-or-later"
 homepage="https://firejail.wordpress.com"
-#changelog="https://raw.githubusercontent.com/netblue30/firejail/master/RELNOTES"
+##changelog="https://raw.githubusercontent.com/netblue30/firejail/master/RELNOTES"
```

Comment:
Ah, this is to force it to push, I'll change it back again.

Reply:
If people made changes to master since you created your branch, you can just rebase to be able to force-push. That way you don't need to change the template.

Reply:
Ah I'll keep that trick in mind. Thanks!